src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  background-color: rgb(214, 214, 214);
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  background-color: #00acdc;
+  height: 4px;
+  width: 37px;
```

Comment:
Consider no hardcoding width, it should have the width of the entire block

Reply:
Fixed.